website/docs/audit.md

```diff
@@ -124,13 +132,22 @@ In addition to violations, these other audit events may be useful (all uniquely
 All of these events (including `violation_audited`) are marked 
 with the same `audit_id` for a given audit run.
 
+### Pubsub channel
+
+This feature uses publish and subscribe (pubsub) model that allows Gatekeeper to export audit violations over a broker that can be consumed by a subscriber independently. Therefore, pubsub violations are not subject to reporting limits. Please refer to [this](pubsub.md) guide to configure audit to push violations over a channel.
+
+Limitations/drawbacks of getting violations using pubsub channel:
+
+- There is an inherent risk of messages getting dropped. You might not receive all the published violations.
+- Additional dependancy on pubsub broker. 
+
 ## Running Audit
 For more details on how to deploy audit and 
 number of instances to run, please refer to [operations audit](operations.md#audit). 
 
 ## Configuring Audit
 
-- Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`)
+- Audit violations per constraint: set `--constraint-violations-limit=123` (defaults to `20`). To get violations using pubsub model, please refer to [this](pubsub.md) guide.
```

Comment:
Please add details for: How does using the pubsub impact this flag? Is the flag ignored or can users increase it or even make it unlimited?